src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -79,16 +81,21 @@ function SplitBillDetailsPage(props) {
                     pointerEvents="box-none"
                     style={[styles.containerWithSpaceBetween]}
                 >
+                    {isScanning && <MoneyRequestHeaderStatusBar />}
                     {Boolean(participants.length) && (
                         <MoneyRequestConfirmationList
                             hasMultipleParticipants
                             payeePersonalDetails={payeePersonalDetails}
                             selectedParticipants={participantsExcludingPayee}
                             iouAmount={splitAmount}
-                            iouComment={splitComment}
                             iouCurrencyCode={splitCurrency}
+                            iouComment={splitComment}
+                            iouCreated={splitCreated}
+                            iouMerchant={splitMerchant}
                             iouType={CONST.IOU.MONEY_REQUEST_TYPE.SPLIT}
                             isReadOnly
+                            receiptPath={transaction.receipt && transaction.receipt.source}
+                            receiptSource={transaction.filename}
```

Comment:
Shouldn't this be the source too?
```suggestion
                            receiptSource={transaction.receipt && transaction.receipt.source}
```

Reply:
I thought it was confusing as well, but the `receiptSource` is the `filename` from the transaction, and the `receiptPath` is the `source` property of the receipt 😅

Reply:
Check how `getThumbnailAndImageURIs` accepts `(path, filename)`. I think we can rename `receiptSource` to `receiptFilename` in `MoneyRequestConfirmationList`?

Reply:
That sounds good to me

Reply:
Done in this [commit](https://github.com/Expensify/App/pull/28710/commits/9b2b9427734d1cc8cf55b31df248d08b6bf18032)